src/tools/crawler/main.rs

```diff
@@ -75,10 +86,34 @@ fn start_logger(default_level: LevelFilter) {
 async fn main() {
     start_logger(LevelFilter::INFO);
     let args = Args::parse();
+    let mut seed_addrs: Vec<SocketAddr> = Vec::new();
 
     // Create the crawler with the given listener address.
     let crawler = Crawler::new().await;
 
+    if let Some(seeds) = args.seed_addrs {
+        seed_addrs.extend_from_slice(&seeds);
+    }
+
+    if let Some(dns_seed) = args.dns_seed {
+        for seed_str in dns_seed {
+            let response = lookup_host(&seed_str);
+
+            if let Ok(response) = response {
+                for address in response.iter() {
+                    seed_addrs.push(SocketAddr::new(*address, ZCASH_P2P_PORT)); // DNS addrs use this port
+                    info!(parent: crawler.node().span(), "DNS seed {} address added: {}", seed_str, address);
+                }
+            } else {
+                error!(parent: crawler.node().span(), "DNS seed {} lookup failed: {}", seed_str, response.err().unwrap());
```

Comment:
Maybe we could kill the process here, so we are aware if this ever happens - otherwise, the mistake might go unnoticed by the user (who might have accidentally entered a wrong domain name)

and the user can be our CI e.g.

Reply:
I'm not convinced to killing process if **any ** seed fails. 
I think that there is quite often use case, when eg. in our CI we'll add 10 DNS seeds (every possible seed). Probably, some of them return the same seeds (as each DNS seed can return many addresses) and it would be ok, when 1 out of 10 would be unavailable - I believe we still would like to crawl the network using remaning 9 seeds.
Constantly changing CI configuration because of one or two unavailable seeds seems to generate more workload to maintain the list than just putting all possible seeds and happily ignore if any of them fails.
Besides, user is still notified by `error` message.